lib/galaxy/util/multi_byte.py

```diff
@@ -1,9 +1,15 @@
-import wchartype
+try:
+    import wchartype
+except ImportError:
+    wchartype = None
 
 from six import text_type
 
 
 def is_multi_byte( chars ):
+    if wchartype is None:
+        message = "Attempted to use galaxy.util.multi_byte but dependency wchartype is unavailable."
```

Comment:
``galaxy.util.multi_byte`` is the module galaxy.util.multi_byte.is_multi_byte is the method.